src/handlers/push_message.rs

```diff
@@ -120,5 +124,30 @@ pub async fn handler(
         Provider::Noop(_) => {}
     }
 
+    // Analytics
```

Comment:
Maybe we could log this dispatch too? In case there are some problems with analytics, we'll be sure that at least we are sending them. What do you think?